### PR TITLE
Create a migration script from gmaven_rules

### PR DESCRIPTION
**Pull request by jin**
From https://github.com/jin/rules_maven/pull/14, adapted for `rules_jvm_external`'s 1.0 release.

### CODE REVIEW

File: scripts/migrate_from_gmaven.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Replace the load statements
+find ./ -type f -name BUILD* -exec sed -i 's/gmaven_rules/rules_jvm_external/g' {} \;
+
+# Replace the loaded gmaven_artifact symbol with maven_artifact
+find ./ -type f -name BUILD* -exec sed -i 's/"gmaven_artifact"/maven_artifact = "artifact"/g' {} \;
```
Comment (dkelmer) [inline scripts/migrate_from_gmaven.sh]: Why not leave it at artifact?

File: scripts/migrate_from_gmaven.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Replace the load statements
+find ./ -type f -name BUILD* -exec sed -i 's/gmaven_rules/rules_jvm_external/g' {} \;
```
Comment (dkelmer) [inline scripts/migrate_from_gmaven.sh]: For all of these commands, is it possible people have these rules in bzl files?

File: scripts/migrate_from_gmaven.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  | cut -d')' -f1 \
+  | cut -d'(' -f2 \
+  | sort \
+  | uniq \
+  | sed 's/^/        /; s/$/,/;'
+
+cat <<-EOF
+    ],
+    repositories = [
+        "https://maven.google.com",
```
Comment (dkelmer) [inline scripts/migrate_from_gmaven.sh]: gmaven couldn't handle repositories other than these but some artifacts did need to rely on additional repositories, right? Maybe add a comment after the copy-paste section to say that they may need to add more repositories?

**Comment by jin** [comment]
Hm, on further thought I think it's worth taking a stab at using buildifier to do this instead.

File: scripts/migrate_from_gmaven.sh (hunk 1 above)
Comment (jin) [reply scripts/migrate_from_gmaven.sh]: done.

File: scripts/migrate_from_gmaven.sh (hunk 2 above)
Comment (jin) [reply scripts/migrate_from_gmaven.sh]: Yes, but I don't think this rule should cover them. It's quite basic, but we can add more flexible features in follow up PRs.

File: scripts/migrate_from_gmaven.sh (hunk 3 above)
Comment (jin) [reply scripts/migrate_from_gmaven.sh]: Good point. Added a note.

**Pull request merged**